baldur/build.gradle

```diff
@@ -0,0 +1,36 @@
+plugins {
+    id 'java'
+    id 'maven'
+    id 'maven-publish'
+    id 'com.jfrog.bintray' version '1.8.4'
+    id 'checkstyle'
+    id 'war'
+}
+
+repositories {
+    jcenter()
+    mavenLocal()
+}
+
+sourceCompatibility = 1.8
+
+group = 'org.opencadc'
+
+version = '1000'
+
+war {
```

Comment:
remove war config and just allow the war file to be named baldur.war

Reply:
Done